plugins/notifications-backend/README.md

```diff
@@ -80,25 +92,14 @@ import { PluginEnvironment } from '../types';
 export default async function createPlugin(
   env: PluginEnvironment,
 ): Promise<Router> {
-  // workaround for creating the database when client is not sqlite
-  const dbClient = await env.database.getClient()
-  dbClient.destroy()
-
-  const catalogClient = new CatalogClient({ discoveryApi: env.discovery });
-  const dbConfig = env.config.getConfig('backend.database');
-  // Following is optional
-  const externalCallerSecret = env.config.getOptionalString(
-    'notifications.externalCallerSecret',
-  );
-
   return await createRouter({
     identity: env.identity,
     logger: env.logger,
     permissions: env.permissions,
     tokenManager: env.tokenManager,
-    dbConfig,
-    catalogClient,
-    externalCallerSecret,
+    database: env.database,
```

Comment:
what is the reason for passing the database?

Reply:
The dynamic plugins call the `createRouter` and provide it with the references. We can not customize it as we did in our implementatino of the `createPlugin`